### PR TITLE
Snapcraft - Env variable group

### PR DESCRIPTION
**Pull request by K-Codemagic**

### CODE REVIEW

File: content/yaml-quick-start/building-a-snap-package.md [hunk 1]
```diff
@@ -58,12 +58,14 @@ snapcraft export-login snapcraft-login-credentials
 ```
 
   You will be asked to enter your Snapcraft account username and password.
-
-2. Base64 encode and encrypt the credentials file. Read more about encrypting sensitive info in Codemagic [here](../building/encrypting/). 
-3. Add the encrypted value to `codemagic.yaml` as an environment variable with the name `SNAPCRAFT_LOGIN_CREDENTIALS`.
+  
+2.Note that binary files (i.e. SNAPCRAFT_LOGIN_CREDENTIALS file) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to environment variables and decoded during the build.
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-snap-package.md:62]:
```suggestion
2. Base64 encode and encrypt the credentials file. Note that binary files (i.e. SNAPCRAFT_LOGIN_CREDENTIALS file) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to environment variables and decoded during the build.
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: content/yaml-quick-start/building-a-snap-package.md (hunk 1 above)
Comment (K-Codemagic) [reply content/yaml-quick-start/building-a-snap-package.md]: I have made the change, can you please recheck

Comment (K-Codemagic) [reply content/yaml-quick-start/building-a-snap-package.md]: Changes are made

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**